### DISCUSSION TITLE
403 error on scrapper but website is accessible on browser

### DISCUSSION THREAD

**Discussion by nomanilyas16** [sparklemotion/mechanize#627] [Q&A]
Hi, I'm trying to access this [website](https://casesearch.courts.state.md.us/casesearch/) but getting 403 error but the same website is accessible with the browser.

Thanks in advance

**Comment by flavorjones** [comment]
It's likely that in your browser you have cookies set by this site (corresponding to the "I agree" button on the public access disclaimer page). Mechanize supports cookies, but when you create a new instance, its cookie jar will be empty (similar to what an incognito browser session has).

Your Mechanize agent should probably click through that initial public access disclaimer page, thereby creating the necessary cookie.

**Reply by nomanilyas16** [reply]
Even the disclaimer page is also throwing 403 error with Mechanize. I have tried it.

**Reply by nomanilyas16** [reply]
<img width="1348" alt="Screenshot 2023-12-05 at 12 14 46 AM" src="https://github.com/sparklemotion/mechanize/assets/66265668/116d457d-eef2-4a83-8365-8ade8e96fed3">
You can check in screenshot that I am trying to get the disclaimer page and it's throwing 403 error

**Reply by flavorjones** [reply]
Well, then that site may be blocking unknown user agents. You may want to try having Mechanize use one of the ones built in, see https://www.rubydoc.info/gems/mechanize/Mechanize#AGENT_ALIASES-constant
